subsys/mpsl/mpsl_fem.c

```diff
@@ -168,14 +163,21 @@ static int fem_nrf21540_gpio_configure(void)
 #else
 			MPSL_FEM_DISABLED_GPIO_CONFIG_INIT
 #endif
-		},
-		.ppi_channels = {
-			CONFIG_MPSL_FEM_NRF21540_GPIO_PPI_CHANNEL_0,
-			CONFIG_MPSL_FEM_NRF21540_GPIO_PPI_CHANNEL_1,
-			CONFIG_MPSL_FEM_NRF21540_GPIO_PPI_CHANNEL_2
 		}
 	};
 
+	for (int i = 0; i < ARRAY_SIZE(cfg.ppi_channels); i++) {
```

Comment:
You could consider creating a static function out of this loop and using the code in both FEM variants, but I'm fine to leave it as it is.

Reply:
Done